README.md

```diff
@@ -1088,6 +1088,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [Braintrust](https://www.braintrustdata.com/) - Evals, prompt playground, and data management for Gen AI. Free plan gives upto 1,000 private eval rows/week.
   * [Findr](https://www.usefindr.com/) - Universal search that lets you search all your apps, at once. Search assistant that lets you answer questions using your information. Free plan offers unlimited unified search and 5 co daily co pilot queries.
   * [ReportGPT](https://report-gpt.io) - AI Powered Writing Assistant. The entire platform is free as long as you bring your own API key.
+  * [Clair](https://askclair.ai/) - Clinical AI Reference. Simply ask and receive. Clair's responses always come with links to original, clinically significant references like FDA Drug Package Inserts, the NIH Clinical Trials Database, the NLM Drug Interactions Database, and Clinical Guidelines. Students have free access to the professional tool suite, which includes Open Search, Clinical Summary, Med Review, Drug Interactions, ICD-10 Codes, and Stewardship. Additionally, a free trial for the professional suite is available.
```

Comment:
please shorten this description, focus on the key points and remove marketing speak words.